src/app/components/group/group.component.ts

```diff
@@ -13,6 +13,7 @@ export class GroupComponent implements OnInit {
   @Input() groupId: string;
   @Input() messages: Message[];
   @Input() enabledGroup: string;
+  @Input() allowAdminFunctions; boolean = false;
```

Comment:
You're not doing anything with this variable in the template of the group component itself. You should probably hide the edit group buttons if this is false.